webpack/components/PaginationRow/index.js

```diff
@@ -0,0 +1,88 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Paginator } from 'patternfly-react';
+
+const defaultPerPageOptions = [5, 10, 15, 25, 50];
+
+const initPagination = (props) => {
+  const pagination = props.pagination || {};
+  const defaultPagination = {
+    page: 1,
+    perPage: 10,
+    perPageOptions: defaultPerPageOptions,
+  };
+  return { ...defaultPagination, ...pagination };
+};
+
+class PaginationRow extends Component {
```

Comment:
Do you mind adding tests for this class?

Reply:
Done